cmd/launch.go

```diff
@@ -260,7 +276,21 @@ func ValidateLaunchFlags(flags *config.SimpleInfo) bool {
 		fmt.Println("Error: You can't define the version without launch template")
 		return false
 	}
-
+	if flags.UserDataFilePath != "" {
+		_, err := os.Stat(flags.UserDataFilePath)
+		if err != nil {
+			fmt.Println("Error: Boot script file path invalid or does not exist")
+			return false
+		}
+	}
+	if flags.UserTags != nil {
+		for _, rawTag := range flags.UserTags { //[tag1:val1, tag2:val2]
+			if len(strings.Split(rawTag, ":")) != 2 { //[tag1,val1]
```

Comment:
typo in tag1,val1 -> tag1:val1

Reply:
this was intentional since `strings.Split` will return a slice without the delimiter

- this will probably be updated anyway given your other comments